src/app.tsx

```diff
@@ -37,9 +37,7 @@ function App(): JSX.Element {
       <div>
         <CustomGlobalHotKeys />
         <HeaderComponent />
-        <div id={"page_without_header"}>
```

Comment:
Przez tą zmianć pojawiał się taki problem. 
![image](https://user-images.githubusercontent.com/21079319/101990653-4e413280-3ca8-11eb-8648-63ac06c44fff.png)

Na razie nie wymyśliłem jak rozwiązać to lepiej, więc zrobiłęm tak. Ale to poprawię później